UnityProject/Assets/Scripts/Objects/PushPull.cs

```diff
@@ -1006,126 +950,102 @@ private void OnServerTileReached(Vector3Int newPos)
+		Logger.LogTraceFormat( "{0} predictive push to {1}", Category.PushPull, pushApproval, serverPos );
 
 		//if predictive lerp is finished
-		if (pushApproval == ApprovalState.Approved)
-		{
-			if (pushPrediction == PushState.Finished)
-			{
+		if ( pushApproval == ApprovalState.Approved ) {
+			if ( pushPrediction == PushState.Finished ) {
 				FinishPrediction();
+			} else if ( pushPrediction == PushState.InProgress ) {
+				Logger.LogTraceFormat( "Approved and waiting till lerp is finished", Category.PushPull );
 			}
-			else if (pushPrediction == PushState.InProgress)
-			{
-				Logger.LogTraceFormat("Approved and waiting till lerp is finished", Category.PushPull);
-			}
-		}
-		else if (pushApproval == ApprovalState.Unexpected)
-		{
+		} else if ( pushApproval == ApprovalState.Unexpected ) {
 			var info = "";
-			if (serverPos == lastReliablePos)
-			{
+			if ( serverPos == lastReliablePos ) {
 				info += $"lastReliablePos match!({lastReliablePos})";
-			}
-			else
-			{
+			} else {
 				info += "NO reliablePos match";
 			}
-			if (pushPrediction == PushState.Finished)
-			{
+			if ( pushPrediction == PushState.Finished ) {
 				info += ". Finishing!";
 				FinishPrediction();
-			}
-			else
-			{
+			} else {
 				info += ". NOT Finishing yet";
 			}
-			Logger.LogFormat("Unexpected push detected OnUpdateRecieved {0}", Category.PushPull, info);
+			Logger.LogFormat( "Unexpected push detected OnUpdateRecieved {0}", Category.PushPull, info );
 		}
 	}
 
 	/// For prediction
-	private void OnClientTileReached(Vector3Int pos)
-	{
-		if (pushPrediction == PushState.None)
-		{
+	private void OnClientTileReached( Vector3Int pos ) {
+		if ( pushPrediction == PushState.None ) {
 			return;
 		}
 
-		if (pos != predictivePushTarget)
-		{
-			Logger.LogFormat("Lerped to {0} while target pos was {1}", Category.PushPull, pos, predictivePushTarget);
-			if (MatrixManager.IsNoGravityAt(pos, false))
-			{
-				Logger.LogTraceFormat("...uh, we assume it's a space push and finish prediction", Category.PushPull);
+		if ( pos != predictivePushTarget ) {
+			Logger.LogFormat( "Lerped to {0} while target pos was {1}", Category.PushPull, pos, predictivePushTarget );
+			if ( MatrixManager.IsNoGravityAt( pos, false ) ) {
+				Logger.LogTraceFormat( "...uh, we assume it's a space push and finish prediction", Category.PushPull );
 				FinishPrediction();
 			}
-			else if (revertPredictivePushHandle == null)
+			else if ( revertPredictivePushHandle == null )
 			{
-				this.StartCoroutine(RevertPushTimer(), ref revertPredictivePushHandle);
+				this.StartCoroutine( RevertPushTimer(), ref revertPredictivePushHandle );
 			}
 			return;
 		}
 
-		Logger.LogTraceFormat("{0} is reached ON CLIENT, approval={1}", Category.PushPull, pos, pushApproval);
+		Logger.LogTraceFormat( "{0} is reached ON CLIENT, approval={1}", Category.PushPull, pos, pushApproval );
 		pushPrediction = PushState.Finished;
-		switch (pushApproval)
-		{
+		switch ( pushApproval ) {
```

Comment:
Codacy found an issue: [Add a 'default' clause to this 'switch' statement.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)